homeworks/anton.nemesh_AntonNemesh/2-dom/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+</head>
+<body>
+    <div class="wrap">
+        <main>
+            <div data-messages></div>
+        </main>
+    </div>
+    <footer>
+        <form action="" data-form onsubmit="return false">
```

Comment:
Весь код обробки івентів краще писати в одному місці щоб не шукати його по всьому апові. Да і хтось може переписати цей хендлер запросто і в тебе буде форма ребутати сторінку. Пренеси це `onsubmit="return false"` всередину `submit` лістенера
```js
function sendMessage(event) {
    event.preventDefault(); // instead of onsubmit="return false"
    //...other your code
}
```